kde-misc/ktoshiba/ktoshiba-5.2.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit kde5
+
+SRC_URI="http://prdownloads.sourceforge.net/${PN}/${PN}-${PV}.tar.xz"
+DESCRIPTION="Function (FN) key monitoring for Toshiba laptops"
+HOMEPAGE="http://ktoshiba.sourceforge.net/"
+LICENSE="GPL-2"
+
+SLOT="5"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug"
```

Comment:
USE=debug is not required as it is set by kde5.eclass, same for SLOT=5, just keep an empty IUSE and remove the slot line altogether ;)